www/src/pages/en/folder-structure.md

```diff
@@ -0,0 +1,281 @@
+   │  └─ index.tsx
+   ├─ styles
+   │  └─ globals.css
+   ├─ types
+   │  └─ next-auth.d.ts
+   ├─ utils
+   │  └─ trpc.ts
+   ├─ pages
+   │  └─ api
+   │     ├─ examples.ts
+   │     └─ restricted.ts
+   ├─ server
+   │  ├─ common
+   │  │  └─ get-server-auth-session.ts
+   │  ├─ db
+   │  │  └─ client.ts
+   │  └─ trpc
+   │     ├─ context.ts
+   │     └─ trpc.ts
+   ├─ pages
+   │  └─ api
+   │     ├─ auth
+   │     │  └─ [...nextauth].ts
+   │     └─ trpc
+   │        └─ [trpc].ts
+   └─ server
+      └─ trpc
+         └─ router
+            ├─ _app.ts
+            ├─ auth.ts
+            └─ example.ts
+├─ .env
+├─ .env-example
+├─ .eslintrc.json
+├─ .gitignore
+├─ README.md
+├─ next-env.d.ts
+├─ next.config.mjs
+├─ package.json
+├─ postcss.config.cjs
+├─ prettier.config.cjs
+├─ tailwind.config.cjs
+├─ tsconfig.json
+```
+
+## `prisma`
+
+The `prisma` folder contains the `schema.prisma` file which is used to configure the database connection and the database schema.
+
+<sub>(With Prisma)</sub>
+
+## `public`
+
+The `public` folder contains static assets that are served by the web server. The `favicon.ico` file is an example of a static asset.
+
+## `src`
+
+The `src` folder contains all the source code of the application.
+
+## `src/env`
+
+Used for environment variable validation and type definitions - see [Environment Variables](/en/usage/env-variables).
+
+### `src/env/client.mjs`
+
+[See the client.mjs documentation](/en/usage/env-variables#servermjs--clientmjs)
+
+### `src/env/server.mjs`
+
+[See the server.mjs documentation](/en/usage/env-variables#servermjs--clientmjs)
+
+### `src/env/schema.mjs`
+
+[See the schema.mjs documentation](/en/usage/env-variables#schemamjs)
+
+## `src/pages`
+
+The `pages` folder contains all the pages of the Next.js application. The `index.tsx` file at the root directory of `/pages` is the homepage of the application. The `_app.tsx` file is used to wrap the application with providers. See [Next.js documentation](https://nextjs.org/docs/basic-features/pages) for more information.
+
+## `src/pages/api`
+
+The `api` folder contains all the API routes of the Next.js application. The `examples.ts` file (with Prisma) contains an example of a route that uses the [Next.js API route](https://nextjs.org/docs/api-routes/introduction) feature along with Prisma. The `restricted.ts` file (with Next-Auth) contains an example of a route that uses the [Next.js API route](https://nextjs.org/docs/api-routes/introduction) feature and is protected by [NextAuth.js](https://next-auth.js.org/).
+
+<sub>(With NextAuth.js, tRPC or tRPC + Prisma)</sub>
+
+## `src/pages/api/auth`
+
+The `auth` folder contains the NextAuth.js authentication slug route.
+
+<sub>(With NextAuth.js)</sub>
+
+### `src/pages/api/auth/[...nextauth].ts`
+
+The `[[...nextauth]].ts` file is the NextAuth.js authentication slug route. It is used to handle authentication requests. See [NextAuth.js usage](/en/usage/next-auth) for more information.
```

Comment:
```suggestion
The `[...nextauth].ts` file is the NextAuth.js authentication slug route. It is used to handle authentication requests. See [NextAuth.js usage](/en/usage/next-auth) for more information.
```

Reply:
Fixed in https://github.com/t3-oss/create-t3-app/pull/731/commits/9a4ac76852359e187ea183f52165628b64e0d26d